Makefile

```diff
@@ -31,4 +45,64 @@ build:
 	operator-sdk bundle create --directory . $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-bundle:$(OPERATOR_BUNDLE_VERSION)
 	docker push $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-bundle:$(OPERATOR_BUNDLE_VERSION)
 	opm index add --bundles  $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-bundle:$(OPERATOR_BUNDLE_VERSION) --tag $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-index:$(OPERATOR_BUNDLE_VERSION) --build-tool=docker
-	docker push $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-index:$(OPERATOR_BUNDLE_VERSION)
\ No newline at end of file
+	docker push $(OPERATOR_REPOSITORY)/$(OPERATOR_IMAGE_NAME)-index:$(OPERATOR_BUNDLE_VERSION)
+
+.PHONY: setup-venv
+## Setup virtual environment for smoke test
+setup-venv:
+	$(Q)python3 -m venv $(PYTHON_VENV_DIR)
+	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade setuptools
+	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade pip
```

Comment:
can you create a folder /scripts/ and have an setup-venv.sh that does
```
	$(Q)python3 -m venv $(PYTHON_VENV_DIR)
	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade setuptools
	$(Q)$(PYTHON_VENV_DIR)/bin/pip install --upgrade pip
```

if needed, the script can take $(Q) , and $(PYTHON_VENV_DIR) as arguments.